counter/templates/counter/producttype_list.jinja

```diff
@@ -4,21 +4,49 @@
   {% trans %}Product type list{% endtrans %}
 {% endblock %}
 
+{% block additional_css %}
+  <link rel="stylesheet" href="{{ static("counter/css/product_type.scss") }}">
+{% endblock %}
+
+{% block additional_js %}
+  <script type="module" src="{{ static("bundled/counter/product-type-index.ts") }}"></script>
+{% endblock %}
+
 {% block content %}
   <p><a href="{{ url('counter:new_producttype') }}">{% trans %}New product type{% endtrans %}</a></p>
```

Comment:
C'est possible de mettre une info box qui dit qu'on peut drag and drop ? Parce que si j'avais pas vu ta petite vidéo j'aurais jamais deviné